### DISCUSSION TITLE
New badges are visible to GitHub staff only

### DISCUSSION THREAD

**Discussion by subidit** [Schweinepriester/github-profile-achievements#489] [General]
Apparently some GitHub staff can see some badges when they are logged in but can't see those badges when they are not. 🥲

Yet to know how to get those badges.

**Comment by Schweinepriester** [comment]
whats the source of the image?

**Reply by subidit** [reply]
GitHub community discussion. 

Update: looks like she removed the image. I don't want her to get into any trouble so removing the image here as well.

**Reply by Schweinepriester** [reply]
thank you for the thread in any case! :)
